src/main/java/frc/robot/subsystems/shooter/commands/SpeedUp.java

```diff
@@ -0,0 +1,28 @@
+package frc.robot.subsystems.shooter.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.shooter.Shooter;
+
+/**
+ * This command changes the velocity of the shooter to a specified velocity, in order to shoot at the best rate.
+ */
+public class SpeedUp extends CommandBase {
+    private final Shooter shooter;
+    private final double velocity;
+
+    public SpeedUp(Shooter shooter, double velocity) {
```

Comment:
I think this command should receive the distance and calculate the velocity based on the vision